includes/Story_Post_Type.php

```diff
@@ -339,9 +366,9 @@ protected static function get_allowed_mime_types() {
 		 * This can be used to add additionally supported formats, for example by plugins
 		 * that do video transcoding.
 		 *
-		 * @since 1.3
-		 *
 		 * @param array Associative array of allowed mime types per media type (image, audio, video).
+		 *
+		 * @since 1.3
```

Comment:
Unrelated change, and also wrong as per https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters

Reply:
PHPCS linter went weird. Fixed https://github.com/google/web-stories-wp/pull/175/commits/ba7d5fc86d484dee22161941caf619a6432f414b